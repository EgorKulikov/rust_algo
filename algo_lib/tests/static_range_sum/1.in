5 5
1 10 100 1000 10000
2 3
0 3
2 5
3 4
0 5
