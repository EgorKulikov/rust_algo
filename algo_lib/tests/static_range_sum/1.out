100
111
11100
1000
11111
